proto/connector_service.proto

```diff
@@ -30,16 +30,25 @@ message SinkParam {
   optional uint32 target_table = 8;
 }
 
-enum SinkPayloadFormat {
-  FORMAT_UNSPECIFIED = 0;
-  JSON = 1;
-  STREAM_CHUNK = 2;
+message JsonFormat{}
+
+message StreamChunkFormat{}
+
+message UnspecifiedFormat{}
+
+message StreamChunkWithSchemaFormat{
```

Comment:
Instead of putting the payload schema in the format message, I think it's better to put the schema field in the `StartSink` message, and when we create the deserializer, we always use the schema field in the `StartSink` message. On rust side, for es sink, when we build the `StartSink` request, we fill in the payload schema field with the special payload schema, and for other sink, we use the original schema of the sink.